src/Stripe.net/Constants/Events.cs

```diff
@@ -866,6 +906,16 @@ public static class Events
         /// </summary>
         public const string TransferCreated = "transfer.created";
 
+        /// <summary>
+        /// Occurs whenever a transfer failed.
+        /// </summary>
+        public const string TransferFailed = "transfer.failed";
+
+        /// <summary>
+        /// Occurs after a transfer is paid. For Instant Payouts, the event will typically be sent within 30 minutes.
+        /// </summary>
+        public const string TransferPaid = "transfer.paid";
+
```

Comment:
Those should not be here. They were deprecated in the API version https://stripe.com/docs/upgrades#2017-04-06 and can never happen but are not properly built internally and still appear in the spec sadly